examples/light-switch-app/silabs/src/LightSwitchMgr.cpp

```diff
@@ -125,6 +125,20 @@ void LightSwitchMgr::TriggerLightSwitchAction(LightSwitchAction action, bool isG
     DeviceLayer::PlatformMgr().ScheduleWork(SwitchWorkerFunction, reinterpret_cast<intptr_t>(data));
 }
 
+void LightSwitchMgr::TriggerLevelControlAction(uint8_t level, bool isGroupCommand)
```

Comment:
The structure of this function doesn't make it easy and straight forward to add new commands.

Reply:
In that case, please remove this function from this PR and add it when it will be used.